ecl/hqlcpp/hqlcpp.cpp

```diff
@@ -1759,6 +1759,7 @@ void HqlCppTranslator::cacheOptions()
         DebugOption(options.expandSelectCreateRow,"expandSelectCreateRow",false),
         DebugOption(options.obfuscateOutput,"obfuscateOutput",false),
         DebugOption(options.showEclInGraph,"showEclInGraph",true),
+        DebugOption(options.showChildCountInGraph,"showChildCountInGraph",false),
```

Comment:
because it isn't very likely to be useful to the users, and have tended to minimize the information in the graph to prevent it becoming too big.